tuna/dbBase/base_class.py

```diff
@@ -71,4 +71,17 @@ def __repr__(self):
     return f"Table name: {self.__table__}\nTable columns: {self.__table__.columns}"
 
 
+class MINIMALBASE():
```

Comment:
What is this class? id is already defined in the BASE class. Since this table will reside in the Tuna DB it needs to inherit BASE. Please remove this class.

Reply:
By inheriting base, the table gets the `valid` column, which doesn't make sense for the analytics. So I created this minimal base class with only the `id` column and no additional stuff. But I can remove it (albeit at the expensive of there being redundant/confusing columns in the solver_analytics_results table)?

Reply:
I can subclass `BASE` from `MINIMALBASE` to get rid of redundant code?